quickstart.org

```diff
@@ -55,57 +82,97 @@ To buy:
    + Mozzarella
 #+END_SRC
 
-Lists can also contain checkboxes: ~[ ]~, ~[-]~, ~[X]~
+Lists can contain checkboxes ~[ ]~, ~[-]~, ~[X]~.
```

Comment:
Do you think it's worth mentioning the meaning associated with these, or is it completely obvious? e.g.
```
~[ ]~ (not started), ~[-]~ in progress, ~[X]~ completed
```
Just a thought

Reply:
Yes will add since I had no idea what the `[-]` form meant.